### PR TITLE
Information page

### PR DESCRIPTION
**Pull request by Md-Almasri**
Hi @eritikass,
I added the header and footer to app file, am I doing right?
Please have a look and let me know if I need to change something, or I need to do it in another way  before I continue.
thanks

### CODE REVIEW

File: src/components/Footer.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<template>
+  <footer>
```
Comment (eritikass) [inline src/components/Footer.vue]: this footer element inside footer component is quite strange (same was also in header), and seems also something that can easily result with errors

File: src/App.vue [hunk 2]
```diff
@@ -1,23 +1,31 @@
...
   </div>
 </template>
 
 <script>
+import Header from './components/Header';
+import Footer from './components/Footer';
+
 export default {
   name: 'app',
+  components: { Header, Footer },
```
Comment (eritikass) [inline src/App.vue]: write object properties to different lines, this also keep git diff in future when you add/remove something more clean - and better to read anycase

```JavaScript
  components: {
    Footer,
    Header,
  },
```

File: src/components/Footer.vue [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+</template>
+
+<script>
+export default {
+  name: 'Footer',
+};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
```
Comment (eritikass) [inline src/components/Footer.vue]: there seems to be styles that go inside each other, what you think about using [SCSS](http://sass-lang.com/)?

```vue
<style lang="scss" scoped>
footer {
  /* .... */

  .footerLogos {
    /* .... */

    .logo {
      /* .... */
    }
  }
}
</style>
```
you need then also add `sass-loader` and `node-sass`
```bash
yarn add -D sass-loader node-sass
```

**Review by eritikass — CHANGES_REQUESTED** [review]
you are in right track for sure - added few comments to to make even better :)

File: src/components/Footer.vue (hunk 1 above)
Comment (Md-Almasri) [reply src/components/Footer.vue]: So should I change footer element into div with footer class name?
or change component name into HeaderComponent?

Comment (eritikass) [reply src/components/Footer.vue]: > So should I change footer element into div with footer class name?

i would go with this one

File: src/components/Header.vue [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+</template>
+
+<script>
+export default {
+  name: 'Header',
+};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
```
Comment (eritikass) [inline src/components/Header.vue]: this also scss (and all other styles)

File: package.json [hunk 5]
```diff
@@ -13,6 +13,8 @@
     "build": "node build/build.js"
   },
   "dependencies": {
+    "node-sass": "^4.7.2",
```
Comment (eritikass) [inline package.json:16]: commit also lock `yarn.lock` change

File: src/components/AddressAndSupporterInfo.vue [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+        <h3>Your new home is located at:</h3>
+        <p>{{address}}</p>
+      </div>
+      <div class="support">
+        <img class="personal-photo" alt="personal photo" v-bind:src="source">
+        <p>{{support}}</p>
+      </div>
+      <div class="btn-container">
+        <button class="btn">Reply to</button>
+        <button class="btn">Show jobs near me</button>
```
Comment (eritikass) [inline src/components/AddressAndSupporterInfo.vue:13]: try to use [Internationalization plugin for Vue.js](https://www.npmjs.com/package/vue-i18n) so your site would be easy to translate

**Comment by Md-Almasri** [comment]
Hi @eritikass ,
Could you please check why personal photo can't be loaded?

**Comment by eritikass** [comment]
He, @Md-Almasri 
because webpack can't understand that you want to bundle image there - for testing you can try something like that

https://github.com/Refugees-Say/information_page/blob/d006760a4742cc0b43c4e38d7d97cfb3387c462c/src/components/AddressAndSupporterInfo.vue#L25
```diff
-      source: '../assets/img_avatar.png',
+      // eslint-disable-next-line
+      source: require('../assets/img_avatar.png'),
```

in the end this image i guess should come from url that are located outside this bundle

**Comment by eritikass** [comment]
@Md-Almasri `LGTM` - merge if you wish, all other improvements can be made as part of separate pull requests

**Comment by Md-Almasri** [comment]
Hi @eritikass 
I'll merge it, and make another branch :)

**Pull request merged**